### PR TITLE
Fixing scope and color of word "constructor"

### PR DESCRIPTION
**Pull request by anubmat**
Fixing issue #32

### CODE REVIEW

File: TypeScript.YAML-tmLanguage [hunk 1]
```diff
@@ -154,15 +154,16 @@ repository:
...
       '1': { name: storage.modifier.ts }
       '2': { name: storage.modifier.ts }
       '3': { name: storage.modifier.ts }
       '4': { name: storage.type.property.ts }
       '5': { name: keyword.operator.ts }
-      '6': { name: entity.name.function.ts }
-      '7': { name: keyword.operator.ts }
+      '6': { name: support.type.ts }
+      '7': { name: entity.name.function.ts }
+      '8': { name: keyword.operator.ts }
```
Comment (DanielRosenwasser) [inline TypeScript.YAML-tmLanguage]: Can you leave a comment on what each of these maps to

**Comment by zhengbli** [comment]
:+1:

**Comment by aeschli** [comment]
@anubmat @zhengbli I think `storage.type` is the better scope (instead of `support.type`)
storage.type describes the type of a element (like var, function, ...)
support.type is a type reference

**Comment by anubmat** [comment]
@aeschli, I will change the scope of 'constructor'  to storage.type with pull request #105. Thanks!

**Pull request merged**